deployment/ansible/roles/nyc-trees.tiler/meta/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+dependencies:
+  - { role: "azavea.nginx", nginx_delete_default_site: True }
+  - { role: "azavea.mapnik" }
+  - { role: "azavea.daemontools" }
+  - { role: "azavea.nodejs" }
```

Comment:
Yeah, it builds without `g++` in place, but it definitely seems to prefer `g++` over whatever the fallback compiler is.  Easy enough to add that role in, will do.